README.md

```diff
@@ -141,6 +141,20 @@ Altium has integrated Git support. The instructions below show how to configure
 - Checklist does not apply to ground and power planes, only to polygons used in routing.
 - Do NOT use overlapping polygons, combine the two polygons together into one and adjust the final shape accordingly.
 
+## Release Checklist
```

Comment:
Should we add a note to make sure DRILL files are generated and present? Will save us the time having to go back after a failed review on the vendor's end.

Reply:
NC Drill files are included in the `gerber` folder output of the output job automatically. They should be viewable in gerbv. I also have a checklist item that says to check that the holes are shown in the PCB.

Reply:
Added to the existing checklist item, that the NC drill file exported must be pulled into gerbv to do this.